src/core/entities/MRPanelEntity.js

```diff
@@ -54,7 +54,8 @@ export class MRPanelEntity extends MRDivEntity {
      * @description Callback function of MREntity - handles setting up this Panel once it is connected to run as an entity component.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?